ibis/backends/pandas/tests/test_cast.py

```diff
@@ -67,7 +68,13 @@ def test_cast_array(t, from_, to, expected):
 @pytest.mark.parametrize(
     ("to", "expected"),
     [
-        ("string", "object"),
+        pytest.param(
+            "string",
```

Comment:
I think here and few of the cases below it'll also apply for `"string[pyarrow]"`